packages/sdk_connectivity_cubit/lib/src/sdk_connectivity_cubit.dart

```diff
@@ -39,7 +39,7 @@ class SdkConnectivityCubit extends Cubit<SdkConnectivityState> {
     }
   }
 
-  Future<void> _connect(String mnemonic, {bool storeMnemonic = false}) async {
+  Future<void> _connect(String mnemonic, {bool storeCredentials = false}) async {
```

Comment:
It seems that on upgrade we won't save the API key(only on restore and register). Why not saving this on each connect?

Reply:
There was an IDE bug and I forgot to revert these changes after refactoring.

API key will be stored on each connect if it has changed or does not exist in secure storage.

Reply:
Done. 
[Store breezApiKey on every connect](https://github.com/breez/misty-breez/pull/196/commits/4feedfc1b115247d5ec15b67aa9a6e7db795515b)

Reply:
and also [simplified _storeBreezApiKey logic](https://github.com/breez/misty-breez/pull/196/commits/7635876d7f0d283716be9f26783ec7b08219962c)